Documentation/helm-values.rst

```diff
@@ -937,6 +937,10 @@
      - Agent container name.
      - string
      - ``"cilium"``
+   * - netfilterCompatMode
+     - Enable the netfilter compatible mode, guarantee that all traffic will pass through netfilter.
```

Comment:
Based on our previous discussion (e.g., see: https://github.com/cilium/cilium/pull/17504#issuecomment-949359594), I believe this will end up confusing end users. I think we should make it clear in the code but also in user-facing documentation that, at least currently, this only affects nodeport traffic.

Reply:
Updated